pymc/pytensorf.py

```diff
@@ -387,7 +386,7 @@ def hessian_diag(f, vars=None):
         return empty_gradient
 
 
-identity = Elemwise(scalar_identity, name="identity")
+identity = tensor_copy
```

Comment:
Nitpick just import it directly in the VI module, no need to define it in pytensorf?

Reply:
I don't think so, but even so we should then deprecate